packages/kbn-ebt-tools/src/performance_metrics/context/performance_context.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { createContext, useMemo, useState } from 'react';
+import { ttfmpPerfomanceMarkers } from '../performance_markers';
+import { useLocation } from 'react-router-dom';
+import afterFrame from '../after_frame';
+
+function measureInteraction() {
+  performance.mark(ttfmpPerfomanceMarkers.startPageChange);
+  const trackedRoutes: string[] = [];
+  return {
+    /**
+     * Marks the end of the page ready state and measures the performance between the start of the page change and the end of the page ready state.
+     * @param pathname - The pathname of the page.
+     */
+    pageReady(pathname: string) {
+      performance.mark(ttfmpPerfomanceMarkers.endPageReady);
+
+      // Time To First Meaningful Paint (ttfmp)
+      if (!trackedRoutes.includes(pathname)) {
+        performance.measure(pathname, {
+          detail: { eventName: 'time_to_first_meaningful_paint' },
```

Comment:
I would actually rename this to be a unique one for Kibana as TTFMP means different things for different applications and several synthetics monitoring tools measures and provides one. 

How about `kibana:time_to_plugin_render_time` or `kibana:plugin_render_time`

Reply:
https://github.com/elastic/kibana/pull/180309/commits/c0f2b2f960ddd88e38068d7234217961f35108ab#diff-533ff799a897d207273d4b8161d1cbc73c72590b0f3e6d9c80fe3410c5dbaac9R25

used `kibana:plugin_render_time`